README.md

```diff
@@ -79,7 +79,7 @@ contra-env-setup/playbooks/group_vars/all/global.yml
 
 ### Minishift setup options
 * setup_minishift: Setup a minishift cluster : default=true
-* minishift_version: Minishift version to use : default=v1.12.0
+* minishift_version: Minishift version to use : default=v1.17.0
```

Comment:
We can't support this as the default until we update the Custom CentOS ISO or we need to change the default CentOS ISO from minishift.  I vote for the former since that will break others.

Reply:
I believe it should work, because the MR also contains the `Workaround for wrong SELinux context` (which was the issue with newer versions of minishift). If we upgrade the CentOS image, then this workaround should no longer be needed.

Reply:
yes, that's main purpose of the 1st patch.
Allow to use v.1.17 with current default ISO.

The commit message even say that it can be reverted later when default ISO have right SELinux labeling